ld: duplicate symbol _info in libxdvi.a(print-dialog.o) and libxdvi.a(font-open.o) for architecture x86_64

Fixed upstream: https://tug.org/svn/texlive?view=revision&revision=61752

--- texk/xdvik/gui/print-log.h.orig
+++ texk/xdvik/gui/print-log.h
@@ -23,7 +23,7 @@
 #ifndef PRINT_LOG_H_
 #define PRINT_LOG_H_
 
-struct save_or_print_info *info; /* forward declaration */
+struct save_or_print_info; /* forward declaration */
 
 /* printlog access functions */
 extern void printlog_create(struct save_or_print_info *info,
